curriculum/challenges/english/25-front-end-development/lecture-working-with-numbers-booleans-and-the-math-object/673271884bf678d8b9c64f56.md

```diff
@@ -14,107 +14,117 @@ Watch the lecture video and answer the questions below.
 
 ## --text--
 
-How can you check if a number is even using the `modulus` operator in JavaScript?
+Which of the following operators should you use to subtract one number from another?
 
 ## --answers--
 
-Use the `modulus` operator and check if the remainder is equal to `0` using strict equality (`===`).
+`+`
 
----
+### --feedback--
 
-Use the `modulus` operator and check if the remainder is greater than `1`.
+Think about the symbol typically used for subtraction.
 
-### --feedback--
+---
 
-Consider what remainder an even number gives when divided by `2`.
+`-`
 
 ---
 
-Use the `modulus` operator and ignore the remainder.
+`*`
 
 ### --feedback--
 
-Consider what remainder an even number gives when divided by `2`.
+Think about the symbol typically used for subtraction.
 
 ---
 
-Use the `modulus` operator and check if the remainder is equal to 1.
+`/`
 
 ### --feedback--
 
-Consider what remainder an even number gives when divided by `2`.
+Think about the symbol typically used for subtraction.
 
 ## --video-solution--
 
-1
+2
 
 ## --text--
 
-Which of these statements best describes operator precedence in JavaScript?
+What’s the output of the following code?
+
+```js
+const result = 4 / 0;
+console.log(result);
+```
 
 ## --answers--
 
-Addition and subtraction have the highest precedence.
+`Infinity`
 
-### --feedback--
+---
 
-Consider which operations are executed first in a mathematical expression.
+`4`
 
----
+### --feedback--
 
-Exponentiation has the highest precedence, followed by multiplication, division, and remainder.
+Remember that it’s impossible to divide by zero, and think about what kind of exception your code will throw in those cases.
```

Comment:
But it's not impossible and it doesn't give an exception, it just doesn't give a finite number as the result is `Infinity`

Maybe a different feedback is needed here?

Reply:
Yeah JS just outputs infinity and throws no exceptions.

Reply:
That's a good point, and it's probably more accurate to say that it's impossible to have an answer.

I'll try to simplify this feedback.